Modules/timemodule.c

```diff
@@ -947,28 +952,44 @@ time_mktime(PyObject *self, PyObject *tup)
     }
 #ifdef _AIX
     /* year < 1902 or year > 2037 */
-    if (buf.tm_year < 2 || buf.tm_year > 137) {
+    if ((buf.tm_year < 2) || (buf.tm_year > 137))  {
         /* Issue #19748: On AIX, mktime() doesn't report overflow error for
          * timestamp < -2^31 or timestamp > 2**31-1. */
         PyErr_SetString(PyExc_OverflowError,
                         "mktime argument out of range");
         return NULL;
     }
+    {
+        time_t clk;
+        int     year = buf.tm_year;
+        int     days = 0;
+
+        /* year < 1970 */
+        if (year < 70) do
```

Comment:
Please, put the braces in the same line as the "if" (https://www.python.org/dev/peps/pep-0007/#id5)

Reply:
Actually, I was wondering if this might be useful for other platforms as well (that cannot handle negative times) - so I put it in additional brackets. I'll use your recommendation and make it "AIX only".